src/main/java/org/folio/service/finance/rollover/LedgerRolloverProgressService.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.service.finance.rollover;
+
+import io.vertx.core.Future;
+import org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverProgress;
+import org.folio.rest.acq.model.finance.LedgerFiscalYearRolloverProgressCollection;
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.exceptions.HttpException;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+
+import java.util.List;
+
+public class LedgerRolloverProgressService {
+
+  private static final String ENDPOINT = "/finance/ledger-rollovers-progress";
+
+  private static final String ENDPOINT_BY_ID = ENDPOINT + "/{id}";
+
+  private final RestClient restClient;
+
+  public LedgerRolloverProgressService(RestClient restClient) {
+    this.restClient = restClient;
+  }
+
+  public Future<List<LedgerFiscalYearRolloverProgress>> getRolloversProgress(String rolloverId, RequestContext requestContext) {
+    String query = "ledgerRolloverId==" + rolloverId;
+    RequestEntry requestEntry = new RequestEntry(ENDPOINT).withQuery(query).withOffset(0).withLimit(1);
```

Comment:
This query already with limit = 1, can we change signaure to return only 1 LedgerFiscalYearRolloverProgress in this case?
Also second method to retrieve progress is not needed in this case

Reply:
Good point, but to avoid critical changes before release I left this method without any changes.